dbt-cta/blocks/models/1_cta_full_refresh/_1_cta_full_refresh__models.yml

```diff
@@ -727,6 +726,32 @@ models:
         tests:
           - unique
 
+  - name: admin_users_base
+    description: ''
+    columns:
+      - name: session
+        description: ''
+      - name: id
+        description: ''
+        tests:
+          - unique
+      - name: email
+        description: ''
+      - name: encrypted_password
+        description: ''
+      - name: _airbyte_ab_id
+        description: ''
+        tests:
+          - unique
+      - name: _airbyte_emitted_at
+        description: ''
+      - name: _airbyte_normalized_at
+        description: ''
+      - name: _airbyte_admin_users_hashid
```

Comment:
Should these have any non-null checks?

Reply:
we removed them because composer was yeeting the tasks because there were too many tests and they were taking too long, RIP